src/ORM/Rule/IsUnique.php

```diff
@@ -53,6 +53,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        $allowMultipleNulls = true;
+        if (isset($options['allowMultipleNulls'])) {
```

Comment:
@dereuromark Sure. But it would be better to discuss it apart from this PR. The above line also can be changed after discussion :)